frontend/rolecall/src/app/api/cast_api.service.ts

```diff
@@ -169,7 +176,7 @@ export class CastApi {
               let uniquePositionIDs = new Set<number>();
               rawCast.subCasts.forEach(val => uniquePositionIDs.add(val.positionId));
               return {
-                uuid: String(rawCast.id),
+                uuid: CastApi.uuidFromRaw(rawCast.id),
```

Comment:
I think the previous approach of `uuid: String(rawCast.id),` was better than adding this indirection since the conversion doesn't require any additional logic.

Reply:
Done.